junit-vintage-engine/src/test/java/org/junit/vintage/engine/discovery/VintageDiscoveryRequestResolverTests.java

```diff
@@ -58,19 +60,21 @@ void logsWarningWhenFilterExcludesClass() {
 			logRecord.getMessage());
 	}
 
-	public static class Foo {
```

Comment:
Since #787 has now been merged this PR should be rebased onto master and this particular change can be reverted.